### ISSUE TITLE
Fatal error using resources

### ISSUE THREAD

**Issue by SamSandq** [frang75/nappgui_src#110]
I wanted to added some resource images to my application, and proceeded as the documentation says:
- added a directory under `.../res/`, giving `.../res/res_images`
- changed the `CMakeLists.txt` file in the main project directory to

```
nap_desktop_app(Funtus "" NRC_EMBEDDED)
target_link_libraries(Funtus ${NAPPGUI_LIBRARIES} "-l sqlite3")
```

- and ran `cmake -G Xcode -S . -B build -DCMAKE_INSTALL_PREFIX=/Users/sam/nappgui`

This FAILED with the following error:

![Screenshot 2024-03-03 at 9 07 30](https://github.com/frang75/nappgui_src/assets/50418806/ab559025-5180-45a8-9875-9df66250bf3e)

I was mystified, but had a hunch that it didn't properly find my files.
And... I changed my source directory from `nappgui projects`to `nappgui_projects`. The result was the following:

![Screenshot 2024-03-03 at 9 27 17](https://github.com/frang75/nappgui_src/assets/50418806/3c59ec5f-d7d4-4161-ac3f-f478aefe6213)

In other words, either `CMake` or `nrc` does not allow spaces in directory names!

Please ascertain which, and correct.

**Comment by frang75** [comment]
NAppGUI projects have never been tested with whitespaces or Unicode characters in path.
